City-of-Helsinki/drupal-hdbt
# UHF-4092: Styling the medium teaser and teaser for news item

# Group news archive [UHF-4092](https://helsinkisolutionoffice.atlassian.net/browse/UHF-4092)
Added some general news styling that can be used on other instances that display news as well.

## What was done
* Styled teaser listing and medium teaser listing for news items.
* Add template for medium teasers and updated the teaser template.

## How to install
* Check instructions from https://github.com/City-of-Helsinki/drupal-helfi-kasvatus-koulutus/pull/147 PR.

## How to test
* Check instructions from https://github.com/City-of-Helsinki/drupal-helfi-kasvatus-koulutus/pull/147 PR.

## Other PRs
* https://github.com/City-of-Helsinki/drupal-hdbt/pull/297
* Here is a PR that is not part of this review but related to it: https://github.com/City-of-Helsinki/drupal-helfi-platform-config/pull/314
